README.md

```diff
@@ -1,56 +1,37 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Hello-Microverse
+In this project I set up the linters and add a html and css file also .gitignore file.The main purpose of this project is to setting up the linters.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages used html and css
+- Frameworks used visual studio code
+- Technologies used Laptop
+   
+   
+   
+# Live Demo (Not available)
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
+To get a local copy up and run git clone git clone git@github.com:SabaAhmad786/hello.git and follow these simple steps.
 
-### Install
+### Open files in visual studio code
 
-### Usage
-
-### Run tests
-
-### Deployment
+### Run the index.html file on any browser
 
 
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub:(https://github.com/SabaAhmad786)
```

Comment:
[OPTIONAL]Please add your twitter username as well as link and also a link to your linkedin if you have one. This is a requirement for all your projects.
- also when adding a link to your github i would recommend
  ` [@SabaAhmad786 ](https://github.com/SabaAhmad786)`
this way, your username is a link to your github profile.

 Kindly Refer to [this link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to learn how to write markup better

It tells potential users of who created the project, and enables potential recruiters contact you easily,